samples/photo_album/README.md

```diff
@@ -1,70 +1,101 @@
-* Changes to the title field are sent in the upload request. This is meant to illustrate the possibility of attaching extra meta-data to each upload image.
-* The upload control uses the `upload_preset` we configured in Configuration step. This uses the settings defined on Cloudinary side to process the uploaded file.
-
-### Unsigned Upload ###
-
-In order to add images to our photo album that would later be retrievable from the Cloudinary service we must select a tag which will serve as our source for the list. In this case `myphotoalbum`.
-While this tag can be set in the upload preset and remain hidden from the client side, in this sample we included it in the request itself to make this sample work without further configuration steps.
-
-### List Resource ###
-
-Cloudinary supports a JSON list resource. 
-This list represents all resources marked with a specific tag during upload (or later through other APIs).
-Whenever a new resource is uploaded with a tag, or an existing resource already tagged is deleted then the list is recalculated. 
-This enables you to group a list of resources which can be retrieved by a single query. The size of the list is currently limited to 100 entires.
-[Learn more](http://cloudinary.com/documentation/image_transformations#client_side_resource_lists)
+Cloudinary React SDK
+=========================
+## About
+The Cloudinary React SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
+
+#### Note
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [React SDK Guide](https://cloudinary.com/documentation/react1_integration).
+
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
+
+## Key Features
+- [Transform](https://cloudinary.com/documentation/react1_video_manipulation#video_transformation_examples) and [optimize](https://cloudinary.com/documentation/react1_image_manipulation#image_optimizations) assets.
+- Generate [image](https://cloudinary.com/documentation/react1_image_manipulation#deliver_and_transform_images) and [video](https://cloudinary.com/documentation/react1_video_manipulation#video_element) tags.
+
+## Version Support
+| SDK Version   | React 15.3 | React 16.2 | React 16.3 | React 17 |
+|---------------|------------|------------|------------|----------|
+| 1.0.0 - 1.0.4 | V          | X          | X          | X        |
+| 1.0.5 - 1.1.4 | X          | V          | X          | X        |
+| 1.2.0 - 1.6.7 | X          | X          | V          | X        |
+| 1.6.8 - 1.7.0 | X          | X          | V          | V        |
```

Comment:
Perhaps for future proofing, we can say 1.6.8 and above? so we won't have to update this readme for every new version.